### ISSUE TITLE
Consider not mentioning "menu button" in "combobox" pattern

### ISSUE THREAD

**Issue by carmacleod** [w3c/aria-practices#1262]
The combobox pattern contains the following prose:

> Note that the listbox and **menu button** patterns are also useful in this scenario; differences between combobox and alternative patterns are described below.

and:

> Two other widgets that are also visually compact and enable users to make a single choice from a set of discrete choices are listbox and **menu button**. One feature that distinguishes combobox from both listbox and **menu button** is that the user's choice can be presented as a value in an editable field, which gives users the ability to select some or all of the value for copying to the clipboard. Comboboxes and **menu buttons** can be implemented so users can explore the set of allowed choices without losing a previously made choice. ...
> ... a **menu button** cannot be marked required, and while it has an accessible name, it does not have a value so is not suitable for conveying the user's choice in its collapsed state.

Question: Do we really want authors to use the menu button pattern as a popup list, including dynamically changing the name of the button as the user makes menu item selections?
I don't know - maybe it actually works in AT and is a valid thing to do? Just feels odd to me.
Opening this issue to have the discussion.

**Comment by JAWS-test** [comment]
A button whose label changes during operation can be found, for example, in the [APG toolbar example](https://www.w3.org/TR/wai-aria-practices-1.2/examples/toolbar/toolbar.html).

I think such an element should not be allowed, or should only be allowed in a few exceptional cases, because the label is only perceptible to all users if no element has been selected. In practice, such elements occur frequently, e.g. in toolbars of rich text editors: For the selection of font and font size, only the currently selected value is displayed, the label results from the context for sighted users. For blind or visually impaired or cognitively impaired people, however, it is difficult to determine the label from the context. It is therefore a violation of WCAG SC 2.4.6 and 3.3.2.

**Comment by mcking65** [comment]
I agree with what @JAWS-test says is problematic with the all too common pattern of changing the name of a menu button to reflect a chosen value in the menu.

And, I think we should change the font menu in the toolbar example into a select-only combobox.

However, if you do not change the name of a menu button to reflect the chosen option, all the statements about menu button in the combobox pattern text are still accurate. I think it better to call out the limitations of the menu button for the single-choice scenario than it is to not mention it. The combobox pattern text points out the limitations. 

The point is to give guidance, and while we avoid anti-patterns, we can still warn about limitations. We could be more explicit about such limitations in a note in the menu button pattern and potentially even mention the WCAG issues referenced by @JAWS-test. We could then reference that note or warning from other patterns or examples.

**Comment by carmacleod** [comment]
> We could be more explicit about such limitations in a note in the menu button pattern ...

This is a good idea.

**Comment by mcking65** [comment]
Changed title of issue and associated project based on @carmacleod's [confirmation of direction on 11/17/2019](https://github.com/w3c/aria-practices/issues/1262#issuecomment-554749547).
